docs/roadmap/phase-12-data-curation-maturity.md

```
@@ -0,0 +1,151 @@
+         "high_repetition": 8,
+         "short_paragraphs": 27
+       },
+       "overall_quality_score": 0.94
+     }
+   }
+   ```
+   ML-based classifiers (NeMo Curator's fastText / DeBERTa quality model) are **out of scope** — deferred to Phase 13+ (model dependency, non-deterministic, reproducibility risk).
+
+5. [ ] **DOCX / Markdown table preservation**
+   `_extract_docx` replaces the current `" | "` flat join with markdown table syntax:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)